packages/story-editor/src/components/mediaPicker/useMediaPicker.js

```diff
@@ -196,7 +197,7 @@ export default function useMediaPicker({
       };
 
       // Create the media frame.
-      const fileFrame = global.wp.media({
+      const fileFrame = window.wp.media({
```

Comment:
What does this change effect?

Reply:
Nothing. They're exactly the same. But webpack replaces `global` with `window` when bundling.

And in webpack v5 it actually deprecates using `global`. You can learn more about that in #5792 

I was just fixing this while being at it.